test/specs/runLwcTests.e2e.ts

```diff
@@ -0,0 +1,416 @@
+      const testingView = await workbench.getActivityBar().getViewControl('Testing');
+
+      // Open the Test Sidebar
+      const testingSideBarView = await testingView?.openView();
+      expect(testingSideBarView).toBeInstanceOf(SideBarView);
+
+      const sidebar = workbench.getSideBar();
+      const sidebarView = sidebar.getContent();
+      const lwcTestsSection = await sidebarView.getSection('LWC TESTS');
+      expect(lwcTestsSection.elem).toBePresent();
+
+      // Click the run test button that is shown to the right when you hover a test class name on the Test sidebar
+      const lwcTestItem = (await lwcTestsSection.findItem('lwc1')) as TreeItem;
+      await lwcTestItem.select();
+      const runTestsAction = await lwcTestItem.getActionButton(
+        'SFDX: Run Lightning Web Component Test File'
+      );
+      if (!runTestsAction) {
+        fail('Could not find run tests action button');
+      }
+      await runTestsAction.elem.click();
+
+      // Verify test results are listed on the terminal
+      // Also verify that all tests pass
+      const terminalText = await utilities.getTerminalViewText(workbench, 15);
+      expect(terminalText).not.toBeUndefined();
+      expect(terminalText).toContain(
+        `PASS  ${path.join('force-app', 'main', 'default', 'lwc', 'lwc1', '__tests__', 'lwc1.test.js')}`
+      );
+      expect(terminalText).toContain('Test Suites: 1 passed, 1 total');
+      expect(terminalText).toContain('Tests:       2 passed, 2 total');
+      expect(terminalText).toContain('Snapshots:   0 total');
+      expect(terminalText).toContain(
+        `Ran all test suites within paths "${path.join(projectFolderPath, 'force-app', 'main', 'default', 'lwc', 'lwc1', '__tests__', 'lwc1.test.js')}`
+      );
+
+      // Verify the tests that are passing are labeled with a green dot on the Test sidebar
+      const icon = await (await lwcTestItem.elem).$('.custom-view-tree-node-item-icon');
+      const iconStyle = await icon.getAttribute('style');
+      // Try/catch used to get around arbitrary flaky failure on Ubuntu in remote
+      try {
+        expect(iconStyle).toContain('testPass');
+      } catch {
+        utilities.log('ERROR: icon did not turn green after test successfully ran');
+      }
+    });
+
+    step('Run Single Test via the Test Sidebar', async () => {
+      utilities.log(`${testSetup.testSuiteSuffixName} - Run Single Test via the Test Sidebar`);
+      const workbench = await (await browser.getWorkbench()).wait();
+      const testingView = await workbench.getActivityBar().getViewControl('Testing');
+
+      // Open the Test Sidebar
+      const testingSideBarView = await testingView?.openView();
+      expect(testingSideBarView).toBeInstanceOf(SideBarView);
+
+      const sidebar = workbench.getSideBar();
+      const sidebarView = sidebar.getContent();
+      const lwcTestsSection = await sidebarView.getSection('LWC TESTS');
+      expect(lwcTestsSection.elem).toBePresent();
```

Comment:
Can this be a helper function?  It's currently duplicated in the following 4 steps:
1. SFDX: Refresh Lightning Web Component Test Explorer
2. Run All tests via Test Sidebar
3. Run All Tests on a LWC via the Test Sidebar
4. Run Single Test via the Test Sidebar